lib/ruby_lsp/requests/definition.rb

```diff
@@ -49,7 +49,13 @@ def initialize(document, index, position, dispatcher, typechecker_enabled)
           node_types: [Prism::CallNode, Prism::ConstantReadNode, Prism::ConstantPathNode],
         )
 
-        target = parent if target.is_a?(Prism::ConstantReadNode) && parent.is_a?(Prism::ConstantPathNode)
+        if target.is_a?(Prism::ConstantReadNode) && parent.is_a?(Prism::ConstantPathNode)
+          target = determine_target(
+            target,
```

Comment:
Just wondering if you have some rule of thumb about when to use one argument per line?